CREDITS.md

```diff
@@ -694,6 +694,20 @@ is licensed under [CC-BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/
 [daneeklu](http://opengameart.org/users/daneeklu) 
 is licensed under [CC-BY-SA 3.0](http://creativecommons.org/licenses/by-sa/3.0/)
 
+* ["Superpowers Asset Packs"](https://github.com/sparklinlabs/superpowers-asset-packs) are licensed under CC0.
+
+* ["Catch Challenger"](https://github.com/alphaonex86/CatchChallenger-datapack) animations are licensed under CC BY 3.0.
+
+* Animations based on [Henrique Lazarini's icons](https://opengameart.org/content/496-pixel-art-icons-for-medievalfantasy-rpg) are under CC0. 
+
+* ["Sparks Animations"](https://opengameart.org/content/sparks-fire-ice-blood) by Clint Bellanger are licensed under CC BY 3.0.

```

Comment:
I understand this would supercede the individual per-technique records for these and other animation packs the animation download script currently generates?

Reply:
Yeah, sorry I lost track of the progress on the per-technique script. With that implemented, I don't think this is required anymore. I'll close it.